src/resources/projects/website/listing/item-grid.ejs.md

```diff
@@ -37,83 +37,101 @@ return !["title", "image", "image-alt", "date", "author", "subtitle", "descripti
+
+<% if (showField('title')) { %>
+`<h5 class="no-anchor card-title listing-title">`{=html} <%= item.title %> `</h5>`{=html}
+<% } %>
+
+<% if (showField('subtitle')) { %>
+`<div class="card-subtitle listing-subtitle">`{=html} <%= item.subtitle %> `</div>`{=html}
+<% } %>
+
+<% if (showField('reading-time')) { %>
+`<div class="listing-reading-time card-text text-muted">`{=html} <%= item['reading-time'] %> `</div>`{=html}
+<% } %>
 
 <% if (fields.includes('categories') && item.categories) { %>
+`<div class="listing-categories">`{=html}
 
-<div class="listing-categories">
-  <% for (const category of item.categories) { %>
-<div class="listing-category" onclick="window.quartoListingCategory('<%=utils.b64encode(category)%>'); return false;"><%= category %></div>
-  <% } %>
-</div>
+<% for (const category of item.categories) { %>
+`<div class="listing-category" onclick="window.quartoListingCategory('<%- utils.b64encode(category ) %>'); return false;">`{=html} <%= category %> `</div>`{=html}
+<% } %>
 
+`</div>`{=html}
 <% } %>
-<% if (showField('description')) { %>
 
-<div class="card-text listing-description delink"><%= item.description %></div>
+<% if (showField('description')) { %>
+`<div class="card-text listing-description delink">`{=html} <%= item.description %> `</div>`{=html}
 <% } %>
-<% 
+
+<%
 const flexJustify = showField('author') && showField('date') ? "justify" : showField('author') ? "start" : "end";
 %>
+
 <% if (showField('author') || showField('date')) { %>
-<div class="card-attribution card-text-small <%-flexJustify%>">
-<% if (showField('author')) { %><div class="listing-author"><%= item.author %></div><% } %>
-<% if (showField('date')) { %><div class="listing-date"><%= item.date %></div><% } %>
-</div>
+`<div class="card-attribution card-text-small <%- flexJustify %>">`{=html}
+
+<% if (showField('author')) { %>
+`<div class="listing-author">`{=html} <%= item.author %> `</div>`{=html}
 <% } %>
 
-<% if (otherFields.length > 0) { %>
+<% if (showField('date')) { %>
+`<div class="listing-date">`{=html} <%= item.date %> `</div>`{=html}
+<% } %>
 
-<table class="card-other-values">
-<% for (const field of otherFields) { 
-let value = readField(item, field);  
-%>
-<tr>
-<td><%= listing.utilities.fieldName(field) %></td>
-<td class="<%-field%>"><%= listing.utilities.outputLink(item, field, value) %></td>
-</tr>
+`</div>`{=html}
 <% } %>
-</table>
 
+<% if (otherFields.length > 0) { %>
+`<table class="card-other-values">`{=html}
+<% for (const field of otherFields) {
+let value = readField(item, field);
+%>
+`<tr>`{=html}
+`<td>`{=html} <%= listing.utilities.fieldName(field) %> `</td>`{=html}
+`<td class="<%- field %>">`{=html} <%= listing.utilities.outputLink(item, field, value) %> `</td>`{=html}
+`</tr>`{=html}
+<% } %>
+`</table>`{=html}
 <% } %>
 
-</div>
+`</div>`{=html}
 <% } %>
 
 <% if (fields.includes('filename') || fields.includes('file-modified')) { %>
 
-<div class="card-footer">
+`<div class="card-footer">`{=html}
```

Comment:
Could be a problem here to wrap in raw inline as our Lua filter does check for div of class `card-footer`
https://github.com/quarto-dev/quarto-cli/blob/247513ca1c072567f44132de6aee3c474c5f5afc/src/resources/filters/modules/dashboard/card.lua#L83-L85

Though from quick search and check, it seems we use Lua processing only in `modules/dashboard/*`, so could be safe in listing page.

Just sharing in case it rings any bells to anyone.